<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtendedtableView</class>
 <widget class="QWidget" name="ExtendedtableView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="savePresetButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../CustomWidgets_images.qrc">
             <normaloff>:/CustomWidgets/Images/_save.png</normaloff>:/CustomWidgets/Images/_save.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="removePresetButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../CustomWidgets_images.qrc">
             <normaloff>:/CustomWidgets/Images/_remove.png</normaloff>:/CustomWidgets/Images/_remove.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="MyTableView" name="myTableView"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTableView</class>
   <extends>QTableView</extends>
   <header>MyTableView/mytableview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../CustomWidgets_images.qrc"/>
 </resources>
 <connections/>
</ui>
